smithy-model/src/main/java/software/amazon/smithy/model/loader/ModelLoader.java

```diff
@@ -73,12 +73,13 @@ static boolean load(String filename, Supplier<InputStream> contentSupplier, Load
     // is then used to delegate loading to different versions of the
     // Smithy JSON AST format.
     //
-    // This loader supports version 0.5.0. 0.4.0 support was removed in 0.10.
+    // This loader supports version 1.0. 0.5.0 and 0.4.0 support was removed in
```

Comment:
Looks like a typo or incomplete thought here?

Reply:
Next line below completes the sentence.  Can this be on one line instead of wrapping?

Reply:
Maybe this be phrased differently as to not have all the numbers right next to each other, likely the source of confusion.

>This loader supports version 1.0. Support for loading versions 0.5.0 and 0.4.0 was removed...

Reply:
Swapped the wording on this to make it more clear.